misc/graylog.conf

```diff
@@ -239,6 +239,8 @@ plugin_dir = plugin
 #
 # ATTENTION: These settings have been moved to the database in 2.0. When you upgrade, make sure to set these
 #            to your previous 1.x settings so they will be migrated to the database!
+#            This setting is only used once on first startup after that 
+#            all settings are made online in “System > Indices” in the Graylog web interface.
```

Comment:
"After that, index related settings can be changed in the Graylog web interface on the "System / Indices" page. Also see http://docs.graylog.org/en/2.3/pages/configuration/index_model.html#index-set-configuration."